src/frontend/src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,232 @@
+    this.purgeValues = [];
+
+    this.sentChart = "line";
+    this.sentValues = [];
+  }
+
+  ngOnInit() {
+    this.getCategories();
+    this.getStatistics();
+    this.getStatisticsHistory();
+  }
+
+  public getCategories(): void {
+    this.configService.getCategories().
+      subscribe(
+      data => {
+        this.congfigurationData = data.categories;
+        console.log("This is the congfigurationData ", this.congfigurationData);
+        this.getCategory();
+      },
+      error => { console.log("error", error) });
+  }
+
+  public getCategory(): void {
+    this.congfigurationData.forEach(element => {
+      this.configService.getCategory(element.key).
+        subscribe(
+        data => {
+          this.categoryData.push(data);
+          console.log("This is the categoryData ", this.categoryData);
+        },
+        error => { console.log("error", error) });
+    });
+  }
+
+  public getStatistics(): void {
+    var labels = [];
+    var values = [];
+    this.statisticsService.getStatistics().
+      subscribe(data => {
+        this.statisticsData = data;
+        console.log("This is the statisticsData ", data);
+        for (var key in data) {
+          values.push(data[key]);
+          labels.push(key);
+        }
+        this.statisticsGraph(labels, values);
+      },
+      error => { console.log("error", error) });
+  }
+
+  statisticsGraph(labels, data): void {
+    console.log("Labels", labels, " data", data);
+    this.data = {
+      labels: labels,
+      datasets: [
+        {
+          label: 'Latest',
+          data: data,
+          backgroundColor: "rgb(176,196,222)"
+        }
+      ]
+    };
+    this.options = {
+      scales: {
+        yAxes: [{
+          ticks: {
+            beginAtZero: false
+          }
+        }]
+      }
+    }
+  }
+
+  public getStatisticsHistory(): void {
+    var labels = [];
+    var values = [];
+    var readingsValues = []
+    var purgeValues = []
+    var sentValues = []
+    this.statisticsService.getStatisticsHistory().
+      subscribe(data => {
+        this.statHistoryData = data.statistics;
+        this.statHistoryData.forEach(element => {
+          Object.keys(element).forEach(aKey => {
+            if (aKey.indexOf("READINGS") !== -1) {
+              readingsValues.push(element[aKey])
+            }
+            if (aKey.indexOf("PURGED") !== -1 && aKey.indexOf("UNSNPURGED") == -1) {
+              purgeValues.push(element[aKey])
+            }
+            if (aKey.indexOf("SENT") !== -1 && aKey.indexOf("UNSENT") == -1) {
+              sentValues.push(element[aKey])
+            }
+          });
+
+        });
+        console.log("This is the history readings ", readingsValues);
+        console.log("This is the history purg ", purgeValues);
+        console.log("This is the history sent ", sentValues);
+        this.historyRGraph([], readingsValues);
+        this.purgeRGraph([], purgeValues);
+        this.sentRGraph([], sentValues);
```

Comment:
For labels arg, please display in range starting from 1 to objects count on X axis